readthedocs/projects/tasks.py

```diff
@@ -571,6 +578,25 @@ def run_build(self, docker, record):
 
         if self.build_env.failed:
             self.send_notifications(self.version.pk, self.build['id'])
+            # send build failure status to git Status API
+            send_external_build_status(
+                version=self.version, build_pk=self.build['id'], status=BUILD_STATUS_FAILURE
+            )
+        elif self.build_env.successful:
+            # send build successful status to git Status API
```

Comment:
These comments aren't useful, they aren't explaining a `why`, but just explaining exactly what the code does, which you can see from reading the code.

Reply:
I'll Update it in #5933